apps/molgenis-components/package.json

```diff
@@ -20,7 +20,7 @@
   "scripts": {
     "dev": "cross-env CLEAR_GEN_FOLDERS=on vite",
     "build": "cross-env SHOW_CASE=off vite build",
-    "build-showcase": "cross-env SHOW_CASE=on vite build ",
+    "build-showcase": "cross-env SHOW_CASE=on vite build && cross-env SHOW_CASE=off vite build",
```

Comment:
= on = off , add commend on why this strange setup is needed , is it?

Reply:
I need 1 command to create both the 'lib' and the 'showCase'. I can take a look if we can more this to gradle to have the two commands. However, in molgenis-viz this approach was used which created this idea.

Reply:
I mean, gradle probably can do a 'run build && run build-showcase' which  might be a nicer way right?

Reply:
yes, maybe we could move the flag into the showcase build config ( instead of passing the ENV , read it from the context)

Reply:
done, refactored to be seperate step